app/controllers/registration_controller.rb

```diff
@@ -9,7 +10,7 @@ def tracksheet
     name     = params[:name] || 'tracksheet'
     sequence = params[:sequence] || 1
     response['content-disposition'] = "attachment; filename=#{name}-#{sequence}.pdf"
-    pdf = help.generate_tracking_pdf(druids)
+    pdf = generate_tracking_pdf(druids)
```

Comment:
And... the tests aren't quite there... https://github.com/sul-dlss/argo/blob/develop/spec/helpers/registration_helper_spec.rb